src/components/Search/Filter.tsx

```diff
@@ -0,0 +1,298 @@
+import React, { ReactElement, useEffect } from 'react'
+import classNames from 'classnames/bind'
+import { addExistingParamsToUrl } from './utils'
+import Button from '@shared/atoms/Button'
+import {
+  FilterByAccessOptions,
+  FilterByTypeOptions
+} from '../../@types/aquarius/SearchQuery'
+import { useRouter } from 'next/router'
+import queryString from 'query-string'
+import styles from './Filter.module.css'
+import { useFilter, Filters, Sort } from '@context/Filter'
+import Input from '@components/@shared/FormInput'
+import Accordion from '@components/@shared/Accordion'
+
+const cx = classNames.bind(styles)
+
+interface FilterStructure {
+  id: keyof Filters
+  label: string
+  type: string
+  options: {
+    label: string
+    value: string
+  }[]
+}
+
+const filterList: FilterStructure[] = [
+  {
+    id: 'serviceType',
+    label: 'Service Type',
+    type: 'filterList',
+    options: [
+      { label: 'datasets', value: FilterByTypeOptions.Data },
+      { label: 'algorithms', value: FilterByTypeOptions.Algorithm }
+    ]
+  },
+  {
+    id: 'accessType',
+    label: 'Access Type',
+    type: 'filterList',
+    options: [
+      { label: 'download', value: FilterByAccessOptions.Download },
+      { label: 'compute', value: FilterByAccessOptions.Compute }
+    ]
+  },
+  {
+    id: 'filterSet',
+    label: 'Categories',
+    type: 'filterList',
+    options: [
+      {
+        label: 'automotive',
+        value: 'automotive'
+      },
+      {
+        label: 'manufacturing',
+        value: 'manufacturing'
+      },
+      {
+        label: 'text analysis',
+        value: 'textAnalysis'
+      },
+      {
+        label: 'finance',
+        value: 'finance'
+      }
+    ]
+  }
```

Comment:
suggestion: for custom filters I would suggest setting up a config file, similar to the address.config file
e.g., `filters.config.js`
Would be easy to quickly change / setup filters for every portal that way